### PR TITLE
Refactor parameter hierarchy in JHU indicator

### PR DESCRIPTION
**Pull request by sgsmob**
### Description
Updates `jhu` `params.json` files in accordance with the structure laid out in #795.

### CODE REVIEW

File: ansible/templates/jhu-params-prod.json.j2 [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-  "static_file_dir": "./static",
-  "export_dir": "/common/covidcast/receiving/jhu-csse/",
-  "cache_dir": "./cache",
-  "base_url": "https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_{metric}_US.csv",
-  "aws_credentials": {
-    "aws_access_key_id": "{{ delphi_aws_access_key_id }}",
-    "aws_secret_access_key": "{{ delphi_aws_secret_access_key }}"
+  "common": {
+    "export_dir": "/common/covidcast/receiving/jhu-csse/",
+    "log_exceptions": false,
```
Comment (chinandrew) [inline ansible/templates/jhu-params-prod.json.j2:4]: this is new in this json, was that intentional?

File: jhu/delphi_jhu/run.py [hunk 2]
```diff
@@ -69,21 +69,20 @@ def run_module():
...
+    export_dir = params["common"]["export_dir"]
+    base_url = params["indicator"]["base_url"]
     logger = get_structured_logger(
-        __name__, filename=params.get("log_filename"),
-        log_exceptions=params.get("log_exceptions", True))
+        __name__, filename=params["common"].get("log_filename"),
+        log_exceptions=params["common"].get("log_exceptions", True))
 
-    if len(params["bucket_name"]) > 0:
+    if params["archive"]:
```
Comment (sgsmob) [inline jhu/delphi_jhu/run.py:79]: This also needs to be switched

File: ansible/templates/jhu-params-prod.json.j2 (hunk 1 above)
Comment (sgsmob) [reply ansible/templates/jhu-params-prod.json.j2:4]: No, removed

**Review by chinandrew — APPROVED** [review]
:shipit::shipit::shipit:

**Pull request merged**